lib/engine/game/base.rb

```diff
@@ -762,6 +762,10 @@ def purchasable_companies(entity = nil)
         end
       end
 
+      def purchasable_unsold_companies
```

Comment:
who uses this method? didn't this change from the definition in step?

Reply:
In general code it is use in stock.rb (view) as one of the criterias if BuyCompaniesAtFaceValue should be rendered.
The BuyCompaniesAtFaceValue is in general code as well.

Default implementation is in lib/engine/game/base.rb and return empty list, which means the BuyCompaniesAtFaceValue is not normally rendered in the Stock round. 
1860 had the situation that it could have unsold purchasable companies, and therefor this was rendered by mistake in that game, causing it to break.

The reason it broke was that the default was to show any unsold purchasable companies, and then override in 1860.

But in 1824 I now thought I needed this method in round/g_1824/stock.rb as the active step is no longer present when stock round is finished.

So I refactored this to put the onus on 1824 to implement the method, and remove the implementation in 1860 (that did override it to return empty list).

If you want I can change it back a bit, putting it in step base instead?  I still think default can be empty list.
1824 can handle it by having it in its game base, and call it from the 1824 step and 1824 stock round, to avoid duplication of implementation.

Reply:
When coming down to your comment about stock I realized that I do get a double loop over companies. So avoiding the duplication is probably worthless....

So I will change it back a bit to move method to step instead. But I still think changing default to [] is good to avoid problems in other titles.